src/integration-test/tests/legal/certificateOfService/steps/certificateOfService.ts

```diff
@@ -0,0 +1,20 @@
+import { WhatDocumentsPage } from '../pages/what-documents'
+import { DocumentUploadPage } from '../pages/document-upload'
+
+const whatDocumentsPage: WhatDocumentsPage = new WhatDocumentsPage()
+const documentUploadPage: DocumentUploadPage = new DocumentUploadPage()
+
+export class CertificateOfServiceSteps {
+  // todo not used, remove?
+  open () {
```

Comment:
Steps objects usually should not have `open` method. Feel free to remove.

Reply:
done